### DISCUSSION TITLE
各位，录视频屡屡碰壁怎么办？在线等，挺急的

### DISCUSSION THREAD

**Discussion by qiuxi903** [share121/inter-knot#1375] [其他]
最近在给我去年录的视频进行重置，按去年的方法和网上的新方法试了一遍，发现都不行，但是预告都发出去了，啊啊啊，我要烦死啦！[https://www.bilibili.com/video/BV1Ch4y1k7Ki/?spm_id_from=333.337.search-card.all.click&vd_source=62b5aa09c247b7fea7b9b6bffbd65ab2](url)
![mcpfp_-_qiu_xi_123](https://github.com/user-attachments/assets/8d0b18b2-05ba-4cdf-b811-6462b4539ee8)

**Comment by gaojiaobei** [comment]
替你悲伤
